_docs-sources/intro/overview/josh-feedback.md

```diff
@@ -0,0 +1,66 @@
+---
+'title': "Josh's Feedback"
+---
+
+# Josh's Feedback
+
+- What's the vision for "tool fundamentals"?
+  - How does it compare to Set Up Your Environment / Installing Neccessary Tools?
+  - Should "Setup Your Tools" (formerly Tool Fundamentals) be part of Set Up Your Environment?
+- "Next Steps" at bottom should be the same boldness. 
```

Comment:
Actually, this is correct as is. Next Steps is an individual article, whereas the rest of the items in this sidebar are categories.